Arelle/Arelle
# Document Popular Plugins and Archive Old Plugins

#### Reason for change
resolves #791

* Add documentation for how to use many of the most commonly used plugins.
* Many of the plugins are broken/untested. Move many of them to an archive directory that can be used from the plugin manager browse dialog instead of cluttering the built-in menu.
* Update Windows installer/uninstaller to clear install directory so archived plugins aren't left behind (this also addresses issues with legacy installs where old python libs are still present).

#### Steps to Test
* Check the new RTD [build for this branch](https://arelle.readthedocs.io/en/xt-3104/plugins/popular/popular_plugins.html)
* With the latest Arelle build enable a plugin that's archived in this PR.
* Update to this branch.
* Launch Arelle and confirm that a warning is logged informing the user that the plugin could not be located and may have been archived.
* Verify that the plugin is no longer enabled.

**review**:
@Arelle/arelle
